test/integration/dss/test_dss_cli.py

```diff
@@ -131,6 +130,7 @@ def test_upload_progress_bar(self):
     @unittest.skipIf(os.name is 'nt', 'No pty support on Windows')
     def test_upload_progress_bar_interactive(self):
         """Tests upload progress bar with a simulated interactive session"""
+        import pty
```

Comment:
Consider adding a comment explaining why this import is in an unusual place.

Reply:
Done.